indico/modules/events/editing/models/tags.py

```diff
@@ -13,6 +13,11 @@
 from indico.util.string import format_repr, return_ascii
 
 
+tag_colors = [
```

Comment:
What about putting this as `get_sui_colors()` in indico.web.forms.colors?

Reply:
good idea!